### ISSUE TITLE
AccName "completion" step is disconnected from the computation list

### ISSUE THREAD

**Issue by cookiecrook** [w3c/accname#142]
This text comes after "Step 2" in the AccName computation, but it's not part of the computation list. Should it be? I was thinking this could be "Step 3" or the named step "Completion." 

> Append the result of each step above, with a space, to the total accumulated text.
> After all steps are completed, the total accumulated text is used as the accessible name or accessible description of the element that initiated the computation.

**Comment by accdc** [comment]
I agree, it should be part of the explicit algorithm for clarity.

**Comment by cookiecrook** [comment]
I will work on this after #186 is merged.
